tools/importer/README.md

```diff
@@ -0,0 +1,50 @@
+## Porting various repositories into mbed-os
+
+importer.py script can be used to import code base from different repositories into mbed-os.
+
+### Pre-requisties
+1. Get the `required` repository clone and update it to commit required. Note: Repository should be placed outside the mbed-os.
+2. Create json file as per template
+
+### JSON file template
+
+You can list all the files and folders which are required to be copied in config file in `file` and `folder` arrays respectively. Script will remove the files/folders specified in `src` section from mbed-os repository before copy operation is performed.
+New commit is created on branch `feature_+repo_name+last_sha` with commit message "[REPO_NAME]: Updated to last_sha"
+
+Note: Only files present in folder will be copied, directories inside the folder will not be copied.
+
+`commit_sha` is list of commits present in mbed-os repo. These commits will be applied after copying files and folders listed above. Each commit is cherry-picked and applied sequentially in list with `-x` option, to record SHA of otriginal commit.
+Note: In case of conflict, you will have to resolve the conflict and make sure appended statement "(cherry picked from commit …​)" is not deleted from commit message. Re-execute the python script to apply rest of the SHA commits.
```

Comment:
I would reword this first sentence into two sentences. Such as:

You must resolve any conflicts that arise during this cherry-pick process. Make sure that the  "(cherry picked from commit ...)" statement is present in the commit message.